src/client/Microsoft.Identity.Client/Platforms/netdesktop/NetDesktopCryptographyManager.cs

```diff
@@ -13,11 +13,16 @@
 using Microsoft.Identity.Client.Utils;
 using Microsoft.Win32.SafeHandles;
 using Microsoft.Identity.Client.Platforms.net45.Native;
+using System.Collections.Concurrent;
 
 namespace Microsoft.Identity.Client.Platforms.net45
 {
     internal class NetDesktopCryptographyManager : ICryptographyManager
     {
+        private static readonly ConcurrentDictionary<string, RSA> s_certificateToRsaMap = new ConcurrentDictionary<string, RSA>();
```

Comment:
@pmaytak - please apply this fix to the other 2 crypto managers involved in confidential client apps: 

https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/e917c1206b48d63fdc46cd8fe81ee20a9654ac4d/src/client/Microsoft.Identity.Client/Platforms/netcore/NetCoreCryptographyManager.cs#L76 (you can also copy the test for this one)

https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/e917c1206b48d63fdc46cd8fe81ee20a9654ac4d/src/client/Microsoft.Identity.Client/Platforms/netstandard13/NetStandard13CryptographyManager.cs#L66

Reply:
Or consolidate the common part of the code in a different file, somewhere in PlatformsCommon for example